Bang/GameEvents/CardEffects/DuelActionHandler.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using Bang.GameEvents.CardEffects.States;
+using Bang.Players;
+using Bang.PlayingCards;
+using Gameplay = Bang.Game.Gameplay;
+
+namespace Bang.GameEvents.CardEffects
+{
+    internal class DuelActionHandler : CardActionHandler
+    {
+        public override HandlerState ApplyEffect(Game.Gameplay gameplay, Player victim, BangGameCard card)
+        {
+            DefenceAgainstDuel response = new DefenceAgainstDuel {Player = victim};
+            
+            return new WaitingBangAfterDuelState(gameplay, victim, gameplay.PlayerTurn){SideEffect = response};
+        }
+    }
+
+    internal class WaitingBangAfterDuelState : HandlerState
```

Comment:
We started to make State classes as separate files. Maybe we should continue this practice? Or change previous files to this template

Reply:
Fixed